README.md

```diff
@@ -69,5 +68,25 @@ service nginx reload
 - activate the virtualenv (`source bin/activate`)
 - run the instructions in the `deploy.sh` file
 - go to `/etc/systemd/system`
-- create a new service with a symlink 
-- reload the daemon 
+- create a new service with a symlink
+- reload the daemon
+
+## Passwords
+
+Access to shared services (OVH, Twitter, etc.) must be done through individual accounts, to which the management is delegated.
+
+However, some systems do not support delegation, or it can sometimes be helpful to get administrator access to master accounts.
+
+A database containing all the necessary passwords is therefore accessible.
+
+### Instructions
+
+* Install [KeePassX](https://www.keepassx.org)
+* Download the [database](openfisca.kdbx)
```

Comment:
Add the implicit step where the user should run KeePassX and follow `Database` > `Open database`?

Reply:
Isn't it up to the user ? I'm just explaining how to get access to the passwords database.

Reply:
We have two steps and only an implicit link between them. If it's a user guide, I would add the third step (run KeePassX and follow Database > Open database). If it's just about where to find the database, it's optional.